org.eclipse.dawnsci.nexus/src/org/eclipse/dawnsci/nexus/NexusConstants.java

```diff
@@ -0,0 +1,143 @@
+/*-
+ * Copyright 2018 Diamond Light Source Ltd.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.eclipse.dawnsci.nexus;
+
+/**
+ * Constants used to deal with NeXus files
+ */
+public class NexusConstants {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)